main.go

```diff
@@ -96,7 +96,7 @@ func usage() {
 }
 
 func runMemphis(s *server.Server) {
-	err := analytics.InitializeAnalytics(s.AnalyticsToken(), s.MemphisVersion(), s.GetCustomDeploymentId())
+	err := analytics.InitializeAnalytics(s.MemphisVersion(), s.GetCustomDeploymentId())
```

Comment:
again you are missing calling close function at the end of the main function

Reply:
I already call close function as in master version